test/raven.test.js

```diff
@@ -2635,6 +2635,62 @@ describe('Raven (public API)', function() {
       });
     });
 
+    it('should respect `ignoreUrls`', function() {
+      this.sinon.stub(Raven, '_send');
+      var TraceKitStub = this.sinon.stub(TraceKit, 'computeStackTrace');
+
+      Raven._globalOptions.ignoreUrls = joinRegExp([/.+?host1.+/, /.+?host2.+/]);
+      TraceKitStub.returns({stack: [{url: 'http://host1/'}, {url: 'http://host1/'}]});
+      Raven.captureMessage('Do not capture');
+      assert.isFalse(Raven._send.called);
+      TraceKitStub.returns({stack: [{url: 'http://host2/'}, {url: 'http://host2/'}]});
+      Raven.captureMessage('Also do not capture');
+      assert.isFalse(Raven._send.called);
+      TraceKitStub.returns({stack: [{url: 'http://host3/'}, {url: 'http://host3/'}]});
+      Raven.captureMessage('Capture!');
+      assert.isTrue(Raven._send.calledOnce);
+    });
+
+    it('should handle empty `ignoreUrls`', function() {
+      this.sinon.stub(Raven, '_send');
+      var TraceKitStub = this.sinon.stub(TraceKit, 'computeStackTrace');
+
+      Raven._globalOptions.ignoreUrls = [];
+      TraceKitStub.returns({
+        stack: [{url: 'http://example.com'}, {url: 'http://example.com'}]
+      });
+      Raven.captureMessage('Capture!');
+      assert.isTrue(Raven._send.calledOnce);
+    });
+
+    it('should respect `whitelistUrls`', function() {
+      this.sinon.stub(Raven, '_send');
+      var TraceKitStub = this.sinon.stub(TraceKit, 'computeStackTrace');
+
+      Raven._globalOptions.whitelistUrls = joinRegExp([/.+?host1.+/, /.+?host2.+/]);
+      TraceKitStub.returns({stack: [{url: 'http://host1/'}, {url: 'http://host1/'}]});
+      Raven.captureMessage('Capture!');
+      assert.equal(Raven._send.callCount, 1);
```

Comment:
You can use `assert.isTrue(Raven._send.calledOnce);` and `assert.isTrue(Raven._send.calledTwice);` to be more consistent with the next test (which uses `callOnce` attribute).